pyshelf/metadata/container.py

```diff
@@ -16,13 +15,6 @@ def __init__(self, bucket_name, cloud_factory, resource_identity, update_manager
         self._yaml_codec = None
         self._bucket_container = None
 
-    def create_wrapper(self, metadata):
```

Comment:
Is this wrapper object ever used?  I like the factory function to always be used so that there is a single place to instantiate it (so we can change the dependencies later).  If its not actually used should we just dump the class all together?